src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java

```diff
@@ -1,617 +1,665 @@
 package org.mariadb.jdbc.internal.stream;
 
-import java.io.*;
-import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
 import java.sql.Timestamp;
 import java.util.Calendar;
 import java.util.zip.DeflaterOutputStream;
 
+import org.mariadb.jdbc.internal.util.ByteBuffer;
 
 public class PacketOutputStream extends OutputStream {
```

Comment:
Okay I understand this point.

I will redo the changement ?

On Fri, Mar 4, 2016 at 11:56 AM, Vladislav Vaintroub <
notifications@github.com> wrote:

> In src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java
> <https://github.com/MariaDB/mariadb-connector-j/pull/72#discussion_r55016598>
> :
>
> >  import java.sql.Timestamp;
> >  import java.util.Calendar;
> >  import java.util.zip.DeflaterOutputStream;
> >
> > +import org.mariadb.jdbc.internal.util.ByteBuffer;
> >
> >  public class PacketOutputStream extends OutputStream {
>
> that was about whitespace changes in the source files. I agree with mike,
> it is not possible to see what really changed , becase the diff is in every
> line
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MariaDB/mariadb-connector-j/pull/72/files#r55016598>.
>